Frontend/ui-driver/src/Screens/SubscriptionScreen/Controller.purs

```diff
@@ -245,7 +278,9 @@ eval (ShowError errorPayload )state = continue state{props{showError = true, sho
 
 eval (LoadAlternatePlans plansArray) state = continue state { data { managePlanData { alternatePlans = alternatePlansTransformer plansArray state}}, props {subView = ManagePlan, showShimmer = false}}
 
-eval (TryAgainButtonAC PrimaryButton.OnClick) state = updateAndExit state { props{showShimmer = true}} $ Refresh
```

Comment:
try refactoring this code, reduce code redundancy here

Reply:
refactored the code

Reply:
sure dude